packages/flutter/test/material/data_table_test.dart

```diff
@@ -565,7 +572,10 @@ void main() {
       home: Material(child: buildTable()),
     ));
     // The `tester.widget` ensures that there is exactly one upward arrow.
-    final Finder iconFinder = find.widgetWithIcon(Transform, Icons.arrow_upward);
+    final Finder iconFinder = find.descendant(
+      of: find.byType(DataTable),
+      matching: find.widgetWithIcon(Transform, Icons.arrow_upward),
+    );
     Transform transformOfArrow = tester.widget<Transform>(iconFinder);
     expect(
       transformOfArrow.transform.getRotation(),
```

Comment:
why did this test need to be changed? does the new transition introduce an arrow_upward?

Reply:
No but the DataTable includes Transforms with that icon, which require the descendant update.